docs/source/get_started/access_info/access_info_css.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>nit: Perhaps redundant, but having a #comment (like in the qss version: "# selecting the IBM Lagos device") or a markdown cell before with brief leading text along the lines: "For example, here is the kind of information you'll be able to retrieve for, say, "ibmq_lagos_qpu""</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>